src/math/hyp1f1.hh

```diff
@@ -360,7 +364,7 @@ namespace rascal {
           s_i = coefficient(i) * izpow;
           if (res > 0 and std::fabs(s_i) < this->tolerance * res and
               n_terms == -1) {
-            this->n_terms = i;
+            // this->n_terms = i;
```

Comment:
away?

Reply:
don't understand this either, felix did this @felixmusil